common/config/rush/command-line.json

```diff
@@ -42,8 +42,8 @@
     },
     {
       "commandKind": "bulk",
```

Comment:
So I was thinking about that but it gets into the problem that the regular npm test will run in watch mode so I think it will not work well. I think just having the `test:ci` should be good enough and then `npm test` the corresponding project where you want to run the test.

Reply:
`test:ci` vs `test` main difference is that 
- it runs in watch mode(for some project)
- jest snapshot will not be able to be updated(in ci mode)

Reply:
Sounds good!